dbfit-java/build.gradle

```diff
@@ -159,6 +159,19 @@ project('hsqldb') {
     }
 }
 
+project('informix') {
+    description = 'DBFit Informix engine'
+
+    dependencies {
+        compile project(':dbfit-java:core')
+        compile 'com.informix:ifxjdbc:4.10.JC4W1@jar'
+    }
+
+    integrationTest.doFirst {
+        dbdeploy('com.informix.jdbc.IfxDriver', 'jdbc:informix-sqli://localhost:16466/dbfit:INFORMIXSERVER=dbfitifserver', 'dbfit', 'DBFIT', 'row')
```

Comment:
Remark similar to tests connection string: can we easily switch to custom test database instance (on another host/port)?